app/lib/services.rb

```diff
@@ -13,9 +13,14 @@ def self.content_store
   end
 
   def self.cached_content_item(base_path)
-    Rails.cache.fetch("finder-frontend_content_items#{base_path}", expires_in: 5.minutes) do
+    content_item = content_store.content_item(base_path)
```

Comment:
You're skipping the cache here - the call to content store needs to be within the `Rails.cache.fetch(...) {}` block otherwise we'll call the content store every time.

Reply:
ok, have changed it, only did this as wasn't quite sure when we want to adhere to the content store ttl